Core/DataObjects/PTMagicData.cs

```diff
@@ -146,6 +146,9 @@ public class MarketTrend
     [DefaultValue("")]
     public string AllowedMarkets { get; set; } = "";
 
+    [DefaultValue(0)]
+    public int IgnoreOutlier { get; set; } = 0;
```

Comment:
Can I suggest a rename as IgnoreOutlier sounds like a boolean value, when it is actually a threshold value. I suggest IgnoreOutlierThreshold

Reply:
How about TrendThreshold?  Would match some of the other MarketTrend nomenclature...